kairon/async_callback/lambda_function.py

```
@@ -0,0 +1,91 @@
+from typing import Text, Dict, Callable
+
+from AccessControl.ZopeGuards import _safe_globals
+from RestrictedPython import compile_restricted
+from RestrictedPython.Guards import safer_getattr
+import orjson as json
+from AccessControl.SecurityInfo import allow_module
+from datetime import datetime, date
+from requests import Response
+from kairon.async_callback.scheduler import add_schedule_job, delete_schedule_job, generate_id
+from kairon.async_callback.mail import send_email
+from functools import *
```

Ruff (0.8.2)

`from functools import *` used; unable to detect undefined names

(F403)